tests/test_git_repo_scanner.py

```diff
@@ -108,9 +123,15 @@ def test_extra_exclusions_get_added(self, mock_load: mock.MagicMock):
     def test_extra_signatures_get_added(self, mock_load: mock.MagicMock):
         mock_load.return_value = (
             self.data_dir / "pyproject.toml",
-            {"exclude_signatures": ["foo"]},
+            {
+                "exclude_signatures": [
+                    {"signature": "foo", "reason": "Reason to exclude signature"}
+                ]
```

Comment:
```suggestion
                "exclude_signatures": (
                    {"signature": "foo", "reason": "Reason to exclude signature"},
                )
```

Reply:
I believe this needs to stay as is because it is mock for data that is loaded from the config file and that data is a list.

Reply:
You're correct. Ignore me :)